### DISCUSSION TITLE
How to use CSS module in create-react-app with Tailwind?

### DISCUSSION THREAD

**Discussion by Shofol** [tailwindlabs/tailwindcss#4618] [Help]
`react-scripts` support CSS modules by default now. But, for using Tailwind we have to use `craco`.  But, `craco `doesn't support CSS module. What should I do to use CSS modules with Tailwind and CRA?

**Comment by dance2die** [comment]
You don't need to use `craco` to run Tailwind CSS.
reference: https://github.com/dance2die/cra-template-tailwindcss-typescript

The caveat is that you need to "wait" (sleep) until Tailwind CSS finishes initial process.  
(or else, you need to refresh the browser every time the dev server starts).

Here is a working demo on CodeSandbox, which uses `react-scripts`, TailwindCSS, and CSS modules.
Demo: https://codesandbox.io/s/react-scripts-tailwind-css-modules-gunxt?file=/src/App.js
